apostrophecms/apostrophe
# Fixes index page pagination localization

## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*
Two devs in Discord mentioned the problem that when they have a piece-type index page with pagination, the links for the subsequent pages are broken due to the lack of a localization prefix. This PR fixes this problem. Closes PRO-3514.

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

1. Spin up a localized site and navigate to an index piece-page with enough pieces to span multiple pages
2. Switch locales and try the links.

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
